utils/borsh.js

```diff
@@ -404,6 +407,9 @@ function borshify (block) {
     Buffer.concat(
       block.next_bps.map((nextBp) =>
         Buffer.concat([
+          nextBp.hasOwnProperty("validator_stake_struct_version") && nextBp.validator_stake_struct_version === 'V2'
```

Comment:
Is the field `validator_stake_struct_version` introduced in the new version?

Reply:
Yes